Add examples of using packit inside a container (#395)
packit/packit.dev: content/docs/guide.md

from our Fedora based image. It contains packit installed from `main` branch,
i.e. the same you'd get by pip installing from Github.

    $ podman run -ti --rm -v $PWD:/src:z quay.io/packit/packit bash
    $ packit
    Usage: packit [OPTIONS] COMMAND [ARGS]...

> Based upon the command you need to performer you should configure your container with the needed 

Reviewer: ```suggestion
Based on the command you need to perform you should configure your container with the needed 
```

> Based on the command you need to perform you should configure your container with the needed 
> secrets and configuration files like in the following examples:

Reviewer 2: ```suggestion
Depending on the command you want to perform you need to mount secrets and
configuration files, like in the following examples:
```

#### packit copr-build

    $ podman run -ti --rm -v ~/.config/copr:/root/.config/copr:z -v $PWD:/src:z quay.io/packit/packit bash
    $ packit copr-build

#### packit build

    $ podman run -ti --rm -v ~/.ssh/:/root/.ssh:z -v $PWD:/src:z quay.io/packit/packit bash
    $ export KRB5CCNAME=/tmp/ccache

>     $ fkinit -u username

Reviewer 2: Sorry, I just noticed this. It would be nice to make it more explicit, something like:

```suggestion
    $ fkinit -u <Fedora Account username>
```

Or if you have a better idea, please use that. The important would be to make it clear that something needs to be substituted here.

>     $ export KRB5CCNAME=/tmp/ccache

Reviewer 2: I've opened packit/packit#1492 :boat: so that this can be dropped in the future.

Author: I saw that, really nice! But what do you mean in the future? Should we wait to release this PR until the image will be available to the users?

Reviewer 2: It's already available, so you could remove this :)

```
$ podman run -it --rm --pull=always quay.io/packit/packit bash -c 'echo $KRB5CCNAME'
...
/tmp/ccache
```

from our Fedora based image. It contains packit installed from `main` branch,
i.e. the same you'd get by pip installing from Github.

    $ podman run -ti --rm -v $PWD:/src:z quay.io/packit/packit bash
    $ packit
    Usage: packit [OPTIONS] COMMAND [ARGS]...

Based upon the command you need to performer you should configure your container with the needed 
secrets and configuration files like in the following examples:

#### packit copr-build

    $ podman run -ti --rm -v ~/.config/copr:/root/.config/copr:z -v $PWD:/src:z quay.io/packit/packit bash
    $ packit copr-build

#### packit build

    $ podman run -ti --rm -v ~/.ssh/:/root/.ssh:z -v $PWD:/src:z quay.io/packit/packit bash

>     $ dnf install krb5-workstation

Reviewer 3: btw, what about adding this package to the image? Do you know how huge is this with all the dependencies?

Author: I thought the same, multiple times while waiting for the command to complete! For me is an improvement. If you do not have any concern with the image being more heavy.

Author: Looking on its size I realized it is already installed. So I can remove that line!

Reviewer 3: Aha..;) So it just built the cache and installed nothing. Nice!

    $ packit
    Usage: packit [OPTIONS] COMMAND [ARGS]...

Based upon the command you need to performer you should configure your container with the needed 
secrets and configuration files like in the following examples:

#### packit copr-build

>     $ podman run -ti --rm -v ~/.config/copr:/root/.config/copr -v $PWD:/src --security-opt label=disable quay.io/packit/packit bash

Reviewer 3: I think you can use the `:z`/`:Z` option for volume instead of `--security-opt label=disable`.

```suggestion
    $ podman run -ti --rm -v ~/.config/copr:/root/.config/copr:z -v $PWD:/src quay.io/packit/packit bash
```

Author: Yes the :z option on mounted volumes is working. Since I copied it from the documentation, do you want me to update also the other examples?

Reviewer 3: Nice! I see you have already updated that.
